src/lib/Components/Home/ArtistRails/ArtistCard.tsx

```diff
@@ -72,11 +71,15 @@ export class ArtistCard extends React.Component<Props, State> {
 
   setFollowStatus(status: boolean): ArtistFollowHandlerResult {
     return new Promise(resolve => {
-      this.setState({
-        following: status,
-        processingChange: false,
-        followStatusChanged: resolve,
```

Comment:
I think we can also remove `followStatusChanged` from `state` altogether, right?